week-4/Homework/mandatory/2-writers.js

```diff
@@ -39,6 +39,8 @@ let writers = [
   }
 ];
 
+message = writers.filter(x => x.alive).forEach(x => console.log(`Hi, my  name is ${x.firstName} ${x.lastName}. I am ${x.age} years old, and work as a ${x.occupation}`))

```

Comment:
You can use map instead of for.each. map is more chain friendly.

Reply:
Yes, map could do as well, even better

Reply:
replacing with map